src/installers/next-auth.ts

```diff
@@ -38,8 +38,12 @@ export const nextAuthInstaller: Installer = async ({
     "src/pages/api/restricted.ts",
   );
 
+  const envSchemaSrc = path.join(nextAuthAssetDir, "env-auth.js");
+  const envSchemaDest = path.join(projectDir, "src/server/env-schema.js");
```

Comment:
you should probably do some more checking to make sure the correct schema is selected. 

as it is now, are you sure that the Prisma-installer will overwrite this and select the `env-prisma-auth.js` _after_ this? if it runs the other way around this will override in the other direction if that makes sense?

_maybe_ having a separate installer for this would make handling this logic easier?

Reply:
I could move it to a separate installer for now. We only have 3 variants, this should be manageable

Reply:
yes, until there are other stuff being made with ast/codeshift stuff there is no reason to do that for this one